cmd/network/show.go

```diff
@@ -40,6 +43,70 @@ const (
 	selParameters
 )
 
+func printEntityNodes(ctx context.Context, stakingConn staking.Backend, registryConn registry.Backend, beaconConn api.Backend, entity *entity.Entity, height int64) error {
+	epoch, err := beaconConn.GetEpoch(ctx, height)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== ENTITY ===\n")
+
+	entityAddr := staking.NewAddress(entity.ID)
+	fmt.Printf("Entity Addr: %s\n", entityAddr.String())
+
+	fmt.Printf("Entity ID:   %s\n", entity.ID.String())
+
+	account, err := stakingConn.Account(
+		ctx,
+		&staking.OwnerQuery{Height: height, Owner: entityAddr},
+	)
+	if err != nil {
+		return err
+	}
+
+	balance := &account.Escrow.Active.Balance
+	commission := account.Escrow.CommissionSchedule.CurrentRate(epoch)
+
+	q := quantity.NewFromUint64(uint64(1_000_000_000))
+	err = balance.Quo(q)
+	cobra.CheckErr(err)
+
+	fmt.Printf("Stake:       %s ROSE\n", balance.String())
+	fmt.Printf("Commission:  %s\n", commission.String())
+
+	fmt.Println()
+	fmt.Printf("=== NODES ===\n")
+	for _, node := range entity.Nodes {
+		fmt.Printf("Node ID: %s\n", node.String())
+		idQuery2 := &registry.IDQuery{
+			Height: height,
+			ID:     node,
+		}
+
+		if nodeStatus, err := registryConn.GetNodeStatus(ctx, idQuery2); err == nil {
+			if node, err2 := registryConn.GetNode(ctx, idQuery2); err2 == nil {
+				fmt.Printf("  Node Roles: %s\n", node.Roles.String())
+				fmt.Printf("  Oasis Core: %s\n", node.SoftwareVersion)
+				fmt.Printf("  Runtimes:\n")
+				for _, runtime := range node.Runtimes {
+					fmt.Printf("    Runtime ID: %s\n", runtime.ID)
+					fmt.Printf("      Runtime Version: %s\n", runtime.Version)
+				}
+				fmt.Printf("  Node Status:\n")
+				fmt.Printf("    Expiration Processed: %t\n", nodeStatus.ExpirationProcessed)
+				fmt.Printf("    Freeze End Time: %d\n", nodeStatus.FreezeEndTime)
+				fmt.Printf("    Election Eligible After: %d\n", nodeStatus.ElectionEligibleAfter)
+			} else {
+				return fmt.Errorf("could not get a node: %s", err2)
+			}
+		} else {
+			fmt.Println("  Node is not active")
+		}
+		fmt.Println()
```

Comment:
It would be ideal if this newline separator between nodes was only printed in case this was not the last node as it currently results in an extra newline at the end.